napari/_vispy/layers/labels.py

```diff
@@ -98,6 +125,49 @@
 """
 
 
+direct_lookup_shade_without_collision = """
```

Comment:
Should also be changed below.

```suggestion
direct_lookup_shader_without_collision = """
```